sdk/scheduler/src/test/java/com/mesosphere/sdk/offer/evaluate/security/CertificateNamesGeneratorTest.java

```diff
@@ -11,18 +10,33 @@
 import org.bouncycastle.asn1.x500.RDN;
 import org.bouncycastle.asn1.x500.style.BCStyle;
 import org.bouncycastle.asn1.x509.GeneralNames;
+import org.bouncycastle.util.encoders.Hex;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
 import org.mockito.Mock;
 import org.mockito.Mockito;
 import org.mockito.MockitoAnnotations;
 
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
 import java.util.*;
 import java.util.stream.Collectors;
 
 public class CertificateNamesGeneratorTest {
 
+    private static final MessageDigest SHA1_HASHER;
+    static {
+        try {
+            SHA1_HASHER = MessageDigest.getInstance("SHA-1");
+        } catch (NoSuchAlgorithmException e) {
+            throw new IllegalStateException(e);
+        }
+    }
+
+    private static final String AUTOIP_TLD = "autoip.tld";
```

Comment:
If these were changed to the defaults, then we would not need the other changes in this test?